analysis-engine/src/main/java/org/sonarsource/sonarlint/core/analysis/container/analysis/AnalysisContainer.java

```diff
@@ -111,7 +111,7 @@ private void addCoreComponents() {
   }
 
   private void addPluginExtensions() {
-    getComponentByType(AnalysisExtensionInstaller.class).install(this, ContainerLifespan.ANALYSIS);
+    getParent().getComponentByType(AnalysisExtensionInstaller.class).install(this, ContainerLifespan.ANALYSIS);
```

Comment:
Contrary to Pico, we can't access components of the current container in the `doBeforeStart()` method (because the container is not ready) so we have to explicitly query the parent container here, because we know that the component is in the parent anyway. But once a container is created, it works as before with inheritance.